lesson2/main.js

```diff
@@ -0,0 +1,17 @@
+// create list property
```

Comment:
【IMO】
You don't need a comment that you can see by looking at the code, so it's better to delete it. 
(I understand the feelings you want to write it.)

Reply:
I search about [comment out in JS](https://techacademy.jp/magazine/22790#ta-toc-3) and that seems to be the recommendation. So I thought you were talking about content of text, is that correct? :)
(I mean it doesn't need comments because we can see what the code means if we see it)